language-plutus-core/.stylish-haskell.yaml

```diff
@@ -19,6 +19,6 @@ steps:
       remove_redundant: false
 
   - trailing_whitespace: {}
-columns: 100
+columns: 120
```

Comment:
@vmchale 120 is a good width, I think. 

You might want to think about only using 2 space indentation, though. (Personally, esp in functional languages, I find 4 spaces just gets me to bump into the right side of the screen too quickly.)